# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/notifications/instance.https.window.js.ini
# Commit: ed71fdc2d78d
# Full Hash: ed71fdc2d78dec085a3d153220bba4aeda89992b
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2025-03-05 04:28:59
# Regressor Bug: 1951379
# File Overlap Count: 1
# Description:
#   Bug 1951379 - Part 3: Add NotificationStorageEntry r=asuth
#   
#   This skips handling `vibrate` for now, as we don't ship it and it also complicates test setup.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D240213
# ==============================================================================

diff -r 61ab7888abc4 -r ed71fdc2d78d testing/web-platform/meta/notifications/instance.https.window.js.ini
--- a/testing/web-platform/meta/notifications/instance.https.window.js.ini	Tue Mar 04 18:00:17 2025 +0000
+++ b/testing/web-platform/meta/notifications/instance.https.window.js.ini	Tue Mar 04 18:00:18 2025 +0000
@@ -1,3 +1,4 @@
+prefs: [dom.webnotifications.requireinteraction.enabled:true]
 [instance.https.window.html]
   expected:
     if (os == "linux") and not tsan and not debug and fission and not asan: [OK, TIMEOUT]